curriculum/challenges/english/04-data-visualization/data-visualization-with-d3/learn-about-svg-in-d3.english.md

```diff
@@ -27,10 +27,10 @@ Add an <code>svg</code> node to the <code>body</code> using <code>append()</code
 tests:
```

Comment:
Could we change the tests to the following? Just seems to be a bit more readable:

```
tests:
  - text: Your document should have 1 <code>svg</code> element.
    testString: assert($('svg').length == 1, 'Your document should have 1 <code>svg</code> element.');
  - text: The <code>svg</code> element should have a <code>width</code> attribute set to 500 or styled to have a width of 500px.
    testString: assert($('svg').attr('width') == '500'||$('svg').css('width') == '500px', 'The <code>svg</code> element should have a <code>width</code> attribute set to 500 or styled to have a width of 500px.');
  - text: The <code>svg</code> element should have a <code>height</code> attribute set to 100 or styled to have a height of 100px.
    testString: assert($('svg').attr('height') == '100'||$('svg').css('height') == '100px', 'The <code>svg</code> element should have a <code>height</code> attribute set to 100 or styled to have a height of 100px.');
```

Reply:
Alright, done